public/src/actions/calls.jsx

```diff
@@ -142,22 +142,16 @@ export function endTwilioCon(userId, campaignId) {
   .catch(err => err);
 }
 
-export function releaseCall(userId, campaignId, callId, currentCallStatus, next = false) {
```

Comment:
did we verify that all usages of `releaseCall` are no longer dependent on the `next` param?

Reply:
Yep